### PR TITLE
Add diagram to CREATE TABLE

### PR DESCRIPTION
**Pull request by bernscode**
Add diagram to CREATE TABLE.

## Info
- **Description**: 
Add diagram to CREATE TABLE.

- **Preview**: 
https://pr-614.d2fbku9n2b6wde.amplifyapp.com/docs/upcoming/sql-create-table/

- **Related code PR**: 
N/A

- **Related doc issue**: 
Resolves no doc issue.

- **Notes**: 
[ Any additional information? ]

## Before merging
  - [ ] (For version-specific PR) I have selected the corresponding software version in **Milestone** and linked the related doc issue to this PR in **Development**.
  - [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`bernscode`, `CharlieSYH`, `emile-00`, & `hengm3467`). 
  - [ ] I have checked the doc site preview, and the updated parts look good. <details><summary>How?</summary>Scroll down and open this link: image
</details>

### CODE REVIEW

**Comment by bernscode** [comment]
@hengm3467 The lines here overlap because I stacked the content inside of the WITH. Should I unstack them to avoid this but make the diagram wider or leave it as is? Thanks!
create-table-overlap

**Comment by hengm3467** [comment]
This looks like a bug. Could you please help take a look ? @Ad-Bean > * https://pr-614.d2fbku9n2b6wde.amplifyapp.com/docs/upcoming/sql-create-table/

**Comment by hengm3467** [comment]
@bernscode  I feel the (column, type, primary key) part is a bit confusing. Our syntax might also need some adjustment. 

image

Not sure if the diagram below will give you some inspiration.

image

**Comment by hengm3467** [comment]
@bernscode Please use the ZeroOrMore('element_name', ',', 'skip') container to indicate optional and repeatable string. The ',' indicate that it's repeated and delimited by ",".

**Comment by hengm3467** [comment]
@bernscode I feel it's either/or relationship for `col_name data_type PRIMARY KEY` and `PRIMARY KEY (col_name data_type)`. I can explain this during our 1on1. Maybe we can use rr.Comments to indicate this as it's not a crucial element (most people would still use the traditional way).

**Comment by hengm3467** [comment]
@bernscode As mentioned, please split the syntax into multiple parts. Let me know if you have any questions.

File: docs/sql/commands/sql-create-table.md [hunk 1]
```diff
@@ -28,6 +28,104 @@ ROW FORMAT data_format
...
+    )
+);
+
+
+<drawer SVG={svg} />
+
+
+
+
+Here is the diagram for the optional WITH clause:
```
Comment (hengm3467) [inline docs/sql/commands/sql-create-table.md]: The second diagram shows more than the WITH clause. Maybe say this is the WITH clause and the rest of the source parameters.

**Review by hengm3467 — APPROVED** [review]
Please implement the comment and merge the request. We have tried out best with the current tool support. We can explore better and easier ways to present syntax. Thanks! @bernscode

**Pull request merged**